src/aiida_wannier90_workflows/utils/workflows/builder/setter.py

```diff
@@ -30,6 +30,8 @@
 from aiida_wannier90_workflows.workflows.projwfcbands import ProjwfcBandsWorkChain
 from aiida_wannier90_workflows.workflows.wannier90 import Wannier90WorkChain
 
+from aiida_hyperqueue.scheduler import HyperQueueScheduler
```

Comment:
Let's move this line into the `set_parallelization`, cause this introduce a dependency on `aiida-hyperqueue`, which is not necessarily installed on a user's machine.

If you want to check whether aiida-hyperqueue is installed or not, use
```python
try:
    import aiida_hyperqueue
    aiida_hq_intalled = True
except ModuleNotFoundError:
    aiida_hq_intalled = False
```